spec/spec_helper_acceptance.rb

```diff
@@ -125,7 +125,7 @@ def device_model
         apply_manifest('include cisco_ios')
         on host, puppet('plugin', 'download', '--server', host.to_s)
         on host, puppet('device', '-v', '--waitforcert', '0', '--user', 'root', '--server', host.to_s), acceptable_exit_codes: [0, 1]
-        on host, puppet('cert', 'sign', '--all'), acceptable_exit_codes: [0, 24]
+        on host, 'puppetserver ca sign --all', acceptable_exit_codes: [0, 24]
```

Comment:
So, depending on the support matrix for this module we probably want to check for the module version and if less than 5.99 do the old way, if more do this way. Here's an example from beaker-puppet that would probably work for you: https://github.com/puppetlabs/beaker-puppet/pull/67/commits/59ec507a5e86bd52592c47cf3edefb650c71f2da#diff-1ad39dbbbad154ea718af70e36bca745R833